dashboard/src/actions/config.test.tsx

```diff
@@ -0,0 +1,33 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import { getType } from "typesafe-actions";
+import actions from ".";
+import Config from "../shared/Config";
+
+const mockStore = configureMockStore([thunk]);
+
+let store: any;
+const CONFIG = "have you tried to turn it off and on again";
```

Comment:
why all caps? why not call this `testConfig` or `mockConfig`?

Reply:
Because of a brain fart due to to many languages. In case of doubt I tend to check this https://github.com/airbnb/javascript#naming--uppercase, in which case is telling me to indeed remove the case. 

Thanks!